docs/bosh-troubleshooting-gcp.adoc

```diff
@@ -0,0 +1,59 @@
+= BOSH Troubleshooting
```

Comment:
i suppose my points were that i don't see it as a troubleshooting activity, and that this content belongs with the bosh director installation lab (that it shouldn't be moved from it)